nym-vpn-desktop/package.json

```diff
@@ -22,7 +22,8 @@
     "fmt": "prettier --write --ignore-unknown \"**/*\"",
     "fmt:check": "prettier --check --ignore-unknown \"**/*\"",
     "tscheck": "tsc --noEmit",
-    "tauri": "tauri"
+    "tauri": "tauri",
+    "gen:licenses": "npx license-checker-rseidelsohn --json --customPath license-fmt.json > public/licenses-js.json"
```

Comment:
what does this do?

Reply:
oh ok I think I get it now

Reply:
It generates the JS licenses data as a json file so `public/licenses-js.json`